### DISCUSSION TITLE
Login/ contact list

### DISCUSSION THREAD

**Discussion by hahndouglasf** [rustdesk/rustdesk#2607] [Q&A]
Hello, I apologize for English, I'm using a translator.

I would like to know how do I configure the login / address book.
I built the application in a Linux Virtual Machine.

Could someone help me please?

Edit1:

![image](https://user-images.githubusercontent.com/91286300/208502160-3b0f0970-a0e5-401c-b4d7-fa16d8965e72.png)

![image](https://user-images.githubusercontent.com/91286300/208502208-4e6e85b6-3c00-4e9c-852c-bf80a0bd5c33.png)

as i mentioned i am using my local vm after i intend to use with domain address.

**Comment by GabroSW** [comment]
I too am looking for guidance on this column
